wdl-engine/src/backend/crankshaft.rs

```diff
@@ -66,6 +67,15 @@ const GUEST_WORK_DIR: &str = "/mnt/work";
 /// The guest path for the command file.
 const GUEST_COMMAND_PATH: &str = "/mnt/command";
 
+/// The guest path for the output directory.
+const GUEST_OUT_DIR: &str = "/workflow_output";
```

Comment:
@peterhuene  - Are these paths not defined somewhere already that can be accessed here?

Reply:
This particular path is the guest path specific to the cleanup operation, so it probably belongs here and can be its own value.